conf/kaust.config

```diff
@@ -0,0 +1,67 @@
+// KAUST Config Profile
+params {
+    config_profile_name = 'KAUST'
+    config_profile_description = 'Profile for use on King Abdullah Univesity of Science and Technology (KAUST) Ibex Cluster.'
+    config_profile_contact = 'Husen Umer (@kaust.edu.sa)'
+    config_profile_url = 'https://docs.hpc.kaust.edu.sa/quickstart/ibex.html'
+    save_reference = false
+    igenomes_ignore = true
+}
+
+// Load genome resources and assets hosted by the Bioinformatics team on IBEX cluster
+includeConfig '/biocorelab/BIX/resources/configs/genomes.yaml'
```

Comment:
@jfy133 the profile does not recognize this statement, do you you have any suggestion how to make it work? 
The purpose is to host a custom set of genome resources (similar to igenomes but locally maintained).

Reply:
I think users would have to specify it with a custom config i.e. with `-c`... @maxulysse any ideas?

Reply:
Unfortauntely this is breaking other pipeliens, so I'm commenting it out ofr now here: https://github.com/nf-core/configs/pull/810